python/cudf/cudf/_lib/stream_compaction.pyx

```diff
@@ -107,6 +108,66 @@ def apply_boolean_mask(list columns, Column boolean_mask):
     return columns_from_unique_ptr(move(c_result))
 
 
+def stable_distinct(list columns,
```

Comment:
Do not add this function. Instead, let's modify the function `unique` below.

Reply:
Please follow my edit here: https://github.com/rapidsai/cudf/pull/11230/files

The function `drop_duplicates` below should also take an additional parameter `preserve_order=False`. When the parameter is `False`, it will call `cudf::distinct(KEEP_ANY)`. If the parameter is `True` then will call `cudf::stable_distinct(kee= input_keep)`.